### PR TITLE
example URL in Footer fixed

### PR DESCRIPTION
**Pull request by pratyushakatti13**
examples URL now opens examples page in new tab.
Issue: #994

### CODE REVIEW

**Review by HADES-01 — APPROVED** [review]
@pratyushakatti13 Great! Thank You.

File: index.html [hunk 1]
```diff
@@ -191,7 +191,7 @@ <h1 class="heading"><i class="fas fa-cloud"></i> sButton CDN</h1>
           <div class="footer-links">
             <a href="https://github.com/sButtons/sbuttons" target="_blank" title="View on Github"> <i
                 class="fab fa-github"></i> GitHub</a>
-            <a href="https://github.com/sButtons/sbuttons" target="_blank" title="View on Github"> <i
+            <a href="https://sbuttons.github.io/sbuttons/examples.html" target="_blank" title="View on Github"> <i
```
Comment (HADES-01) [inline index.html:194]: @pratyushakatti13 Just one thing, Can you remove the target attribute.

**Review by HADES-01 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (pratyushakatti13) [reply index.html:194]: Yeah sure. It should be removed for only 'Examples' link or other 2 Links in Footer?

Comment (HADES-01) [reply index.html:194]: Just for the `Examples`

Comment (pratyushakatti13) [reply index.html]: Its done!

**Review by HADES-01 — APPROVED** [review]
@pratyushakatti13 Great! Thanks.

**Pull request merged**